cli/src/lib/env-builder.ts

```diff
@@ -0,0 +1,79 @@
+import { BlockExplorerClient } from "./block-explorer-client.js";
+import type { Network } from "./constants.js";
+import { GithubClient } from "./github-client.js";
+import { RpcClient } from "./rpc-client.js";
+
+export class EnvBuilder {
```

Comment:
This class holds all the configuration and builds the clients to interact with other systems.